rodeo/src/rodeo/06-either-and-try.scala

```diff
@@ -0,0 +1,186 @@
+package rodeo
+import zio.test._
+
+object EitherAndTry extends Chapter {
+
+  import scala.util.{Try, Success, Failure}
+
+  // The `Try[A]` type is similar to `Option[A]`. It is used to represent the type of a computation that attempts to return an instance of type A.
+  // If the computation succeeds we obtain an instance of type `Success[A]`. If it fails we obtain a wrapped instance of the Throwable class. This instance will have type `Failure`.
+  // The Throwable class is the least derived class of the Scala exception hierarchy. An instance of this class can represent an error or an exception.
+  // For practical purposes the main difference between `None` and `Failure` is that `Failure` is able to provide insight into the cause of the "failure".
+
+  Exercise("Try constructor") {
+    assertTrue(Try(7) == Success(???))
+
+    // Delete the following line if you can't find the answer.
```

Comment:
I suggest to remove this comment, I feel we should write doable exercises, if we think its too hard,  we should give some hints perhaps or rethink the exercise